README.md

```diff
@@ -227,6 +227,36 @@ Software is installed in miniconda environments.  You can use "[conda list](http
 ### How can I contribute to existing Open-Source Fornax notebook tutorials?
   * open issue or PR on Fornax Github [repo](https://github.com/nasa-fornax/fornax-demo-notebooks)
 
+## Using Fornax Software Environments Outside Fornax
+The Fornax software environments are available as docker container images. To use them on your computer, you will need to install
```

Comment:
Should we add an example motivation?  

"If you want to run the Fornax software Environment outside of the Fornax Science Console, that is an available option.  Possible use cases for this are if you want to use different CPU/GPU/RAM/storage options than are available to you for free on the Fornax system.  Possibly you have your own compute or your own cloud based account"  The advantage of using our Software environment is that it is maintained already for you with widely used Astronomy software.